quickstarts/Function_calling.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
         "<table align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb\"><img src=\"../images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb\"><img src=\"https://github.com/Giom-V/gemini-api-cookbook/blob/function_calling/images/colab_logo_32px.png?raw=1\" />Run in Google Colab</a>\n",
```

Comment:
Use relative links here. We don't to break this guide if/when you delete your branch (and relative links should work across forks too)